hack/release.sh

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env bash
+# Copyright 2017 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -euo pipefail
+
+BUILD_DATE="$(TZ=Etc/UTC date +%Y%m%d)"
+RELEASE_DIR="${RELEASE_DIR:-nightly/${BUILD_DATE}}"
+
+SCRIPT_ROOT="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+BUILD_SCM_HASH="$(git rev-parse HEAD)"
+
+TMPDIR="$(mktemp -d /tmp/crrelXXXXXXXX)"
+function clean_up() {
+  if [[ "${TMPDIR}" == "/tmp/crrel"* ]]; then
+    rm -rf "${TMPDIR}"
+  fi
+}
+trap clean_up EXIT
+
+cd "${SCRIPT_ROOT}/.."
+
+# Build copy the tools.
+bazel build //cmd/crinit //cmd/clusterregistry
+
+# Create the archives.
+pushd bazel-bin/cmd/crinit
+tar czf "${TMPDIR}/clusterregistry-client.tar.gz" crinit
```

Comment:
Would `tar czf "${TMPDIR}/clusterregistry-client.tar.gz" -C bazel-bin/cmd/crinit crinit` work instead of `pushd`/`popd` headache? What bothers me about `pushd`/`popd` are the errors that leave the users in a `bazel-bin/...` directory.

Reply:
Yes it does!